pkg/controllers/raycluster_webhook.go

```diff
@@ -132,3 +142,32 @@ func (r *rayClusterDefaulter) Default(ctx context.Context, obj runtime.Object) e
 
 	return nil
 }
+
+func (v *rayClusterValidator) ValidateCreate(ctx context.Context, obj runtime.Object) (admission.Warnings, error) {
+	raycluster := obj.(*rayv1.RayCluster)
+	var warnings admission.Warnings
+	var allErrors field.ErrorList
+	specPath := field.NewPath("spec")
+
+	if *raycluster.Spec.HeadGroupSpec.EnableIngress {
```

Comment:
`pointer(raycluster.Spec.HeadGroupSpec.EnableIngress, false)`

Reply:
Nice one thanks! - I used `pointer.BoolDeref(raycluster.Spec.HeadGroupSpec.EnableIngress, false)` to avoid nil pointers.